.env.example

```
@@ -0,0 +1,5 @@
+OPENAI_API_KEY=
+POSTHOG_API_KEY=
+POSTHOG_HOST=https://us.i.posthog.com
+NEXT_PUBLIC_POSTHOG_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_KEY key should go before the OPENAI_API_KEY key

(UnorderedKey)